villainary-react/app/products/ProductCard.tsx

```diff
@@ -0,0 +1,37 @@
+'use client';
+
+import { Grid, Typography } from '@mui/material';
+import { theme } from '../Theme';
+import Money from '../components/Money';
+import { Currencies } from '../enums/currencies';
+import { Product } from '../interfaces/Product';
+
+type ProductCardProps = {
+    product: Product;
+};
+
+export default function ProductCard(props: ProductCardProps) {
```

Comment:
Looks fine for now.  I'm assuming there will be a task to visit individual product pages and these cards will become clickable - may need hover effects in the future.